src/tinuous/__main__.py

```diff
@@ -676,7 +680,7 @@ class DataladConfig(NoExtraModel):
 
 class Config(NoExtraModel):
     repo: str
-    path_prefix: Optional[str] = None
+    vars: Dict[str, str] = Field(default_factory=dict)
```

Comment:
note to myself -- `dict` is ordered in any sensible python3 version so order of `vars` specification matters and could be "relied upon" I guess.